### PR TITLE
test(travis): Avoid crashing when things hiccup

### PR DESCRIPTION
**Pull request by Droogans**
Saw some failures recently and I think I know what might be causing it.

### LGTM's
- [ ] Dev LGTM
- [ ] Zoom LGTM

### CODE REVIEW

**Comment by Droogans** [comment]
We got something hanging here instead of crashing. Don't think this works...

**Comment by Droogans** [comment]
This isn't right... https://travis-ci.org/rackerlabs/helix-ui/builds/360291935#L661

I need to write some kind of backoff in this PR....

File: docs/_templates/partials/after_footer.njk [hunk 1]
```diff
@@ -8,7 +8,7 @@
       elAdapter.parentElement.removeChild(elAdapter);
     }
   </script>
-  <script src="https://ab2b55b0a26bb32fdc01-2c8b94bcec8e01c366189541a4035ea4.ssl.cf5.rackcdn.com/custom-elements-es5-adapter.js"></script>
+  <script src="vendor/scripts/custom-elements-es5-adapter.js"></script>
```
Comment (Droogans) [inline docs/_templates/partials/after_footer.njk]: Can this be tracked by a package manager? I assume these checked-in scripts will be regenerated frequently without the aid of one.

Comment (CITguy) [reply docs/_templates/partials/after_footer.njk]: These scripts aren't regenerated, just copied. And the ES5 adapter is highly unlikely to change over time.

Comment (CITguy) [reply docs/_templates/partials/after_footer.njk]: I'm gonna see if 6bd77f7 will fix things by itself.

**Comment by CITguy** [comment]
So, we know that by adding the polyfills to the project (rather than relying on the CDN) _fixes_ the issue. Do we want to use this PR to perform further debugging into _what the issue actually is_?

**Comment by Droogans** [comment]
See #188

**Pull request closed without merging**